UITests/TransferMethod/AddTransferMethodBankAccountTests.swift

```diff
@@ -20,21 +20,61 @@ class AddTransferMethodTests: BaseTests {
 
         waitForNonExistence(spinner)
 
-        addTransferMethod.setBranchId(branchId: "021000021")
-        addTransferMethod.setAccountNumber(accountNumber: "12345")
-        addTransferMethod.selectAccountType(accountType: "Checking")
+        addTransferMethod.setBranchId("021000021")
+        addTransferMethod.setAccountNumber("12345")
+        addTransferMethod.selectAccountType("Checking")
+        addTransferMethod.selectRelationship("Self")
+        addTransferMethod.setNameFirst("John")
+        addTransferMethod.setNameLast("Smith")
+        addTransferMethod.setNameMiddle("Adam")
+        addTransferMethod.setPhoneNumber("+16045555555")
+        addTransferMethod.setMobileNumber("+16046666666")
+        addTransferMethod.setDateOfBirth(yearOfBirth: "1981", monthOfBirth: "December", dayOfBirth: "25")
+        addTransferMethod.selectCountry("United States")
+        addTransferMethod.setStateProvince("Maine")
+        addTransferMethod.setStreet("632 Broadway")
+        addTransferMethod.setCity("Bangor")
+        addTransferMethod.setPostalCode("04401")
+
+        app.scrollToElement(element: addTransferMethod.createTransferMethodButton)
+
```

Comment:
@ekan-hw 
1. Task says "Verify that an account can be created". 
Is this implementation enough to fulfill the requirement or more checks should be performed?
2. So far to switch between Individual and business profile I replace token in `IntegratorAuthenticationProvider`. 
Could you please suggest more strict way?

Reply:
You need to create 2 different tests 1 for Individual and 1 for business and mock individual and business jwt token for those 2.

Reply:
@ekan-hw can explain you further in detail

Reply:
Tests updated to use individual and business profile types